docs/release-notes/release-notes-0.19.0.md

```diff
@@ -69,6 +69,12 @@
 
 ## RPC Updates
 
+* Some RPCs that previously just returned an empty response message now at least
```

Comment:
I wonder if there's an easy way to let the existing users know which responses are changed 🤔

Reply:
No easy way, no. But added a manual summary to the release notes.

Reply:
Very nice thanks!